editor-components/pricing-table/pricing-table7/pricing-table7.tsx

```diff
@@ -2,479 +2,1083 @@ import * as React from "react";
 import { BasePricingTable } from "../../EditorComponent";
 import styles from "./pricing-table7.module.scss";
 import ComposerLink from "../../../../custom-hooks/composer-base-components/Link/link";
+import { ComposerIcon } from "../../../composer-base-components/icon/icon";
+import { Margin } from "@mui/icons-material";
+import { useState, useEffect } from 'react';
+import { number } from "yup";
+
```

Comment:
```suggestion
import { ComposerIcon } from "../../../composer-base-components/icon/icon";
import { useState, useEffect } from 'react';
```

check those imports necessary